docs/project/contribution_tools.md

```diff
@@ -199,6 +200,15 @@ These commands will help diagnose potential build issues because they'll expose
 what's occurring with
 [clang detection](/bazel/cc_toolchains/clang_configuration.bzl).
 
+### System development libraries (zlib, etc.) <a name="system-development-libraries"></a>
```

Comment:
There is no need for the anchor here, and you should use the [pre-commit](/docs/project/contribution_tools.md#pre-commit) tool to automatically generate the table-of-contents rather than doing it yourself. It'll handle the linking and formatting for you.

That'll also fix the GH action run.